src/polydb.jl

```diff
@@ -0,0 +1,376 @@
+module Polydb
+
+import Polymake: call_function
+
+using Polymake
+
+# julia itself also has a cert.pem but this one should be more recent
+# and provides a variable for the path
+using MozillaCACerts_jll
+
+using Mongoc
+
+import Mongoc: find
+
+#Polymake.Polydb's types store information via
+# a corresponding Mongoc type variable
+struct Database
+   mdb::Mongoc.Database
+end
+
+struct Collection{T}
+   mcol::Mongoc.Collection
+end
+
+struct Cursor{T}
+   mcursor::Mongoc.Cursor{Mongoc.Collection}
+end
+
+# connects to the Polydb and
+# returns a Polymake.Polydb.Database instance
+#
+# the uri of the server can be set in advance by writing its `String` representation
+# into ENV["POLYDB_TEST_URI"]
+# (used to connect to the github services container for testing)
+function get_db()
+   # we explicitly set the cacert file, otherwise we might get connection errors because the certificate cannot be validated
+   client = Mongoc.Client(get(ENV, "POLYDB_TEST_URI", "mongodb://polymake:database@db.polymake.org/?authSource=admin&ssl=true&sslCertificateAuthorityFile=$(cacert)"))
+   return Database(client["polydb"])
+end
```

Comment:
if this function is used only in testing infrastructure I suggest to define it in the test suite, or at least rename to
`__get_test_db()`

Reply:
actually, this method is used in both normal and testing environment. the `String` passed to the `Mongoc.Client` constructor is determined by whether `ENV["POLYDB_TEST_URI"]` is set. If it is (e.g. in the CI tests), we read the uri from said environment variable. if it is not, we use the uri of the live database